src/SWP/Bundle/ContentBundle/Resources/config/services.yml

```diff
@@ -107,3 +108,27 @@ services:
         parent: "router.symfony"
         tags:
             - { name: router, priority: 300 }
+
+    swp_content_bundle.valid_route_id:
+        class: SWP\Bundle\ContentBundle\Validator\Constraints\RouteIdValidator
+        arguments:
+            - '@swp.provider.route'
+        tags:
+            - { name: validator.constraint_validator, alias: route_in_database }
+
+    swp_content_bundle.routetoarticle.repository:
```

Comment:
http://superdesk-web-publisher.readthedocs.io/en/latest/bundles/SWPStorageBundle/usage.html#register-configured-classes-in-your-extension-class